server/server.js

```diff
@@ -7,13 +7,74 @@ const bcrypt = require('bcryptjs');
 const redis = require('connect-redis')(session);
 const flash = require('connect-flash');
 
+const User = require('../database/models/User');
+
 const PORT = process.env.PORT || 8080;
 const ENV = process.env.NODE_ENV || 'development';
+const SESSION_SECRET = process.env.SESSION_SECRET || 'kabob';
 
 const app = express();
 
 app.use(bodyParser.urlencoded());
 app.use(bodyParser.json());
+app.use(
+  session({
+    store: new redis({ url: process.env.REDIS_STORE_URI, logErrors: true }),
```

Comment:
This is good but should probably add a check that the environment var exists first before passing it to the redis constructor